docs/includes/guides/gradle-build.adoc

```diff
@@ -21,7 +21,7 @@ ifndef::rootdir[:rootdir: {docdir}/../..]
 == Introduction
 
 While most of Helidon's examples use Maven, you can also use Helidon
-with a Gradle project. We recommend Gradle 7+.
+with a Gradle project. We recommend Gradle 8.4+.
```

Comment:
It looks like Gradle 8.4 is required for Java 21 support. So let's make this statement stronger: 

"Gradle 8.4+ is required to build Helidon 4 projects."

Otherwise looks good!